tensorboard/plugins/debugger_v2/debugger_v2_plugin_test.py

```diff
@@ -151,6 +154,33 @@ def testPluginIsActiveWithDataExists(self):
         _generate_tfdbg_v2_data(self.logdir)
         self.assertTrue(self.plugin.is_active())
 
+    def testConcurrentCallsToPluginIsActiveWhenNotActive(self):
+        results = []
+
+        def query_is_active():
+            results.append(self.plugin.is_active())
```

Comment:
It's best not to reply on the built in python data structures to be thread safe (even if some of their operations happen to be for cpython).  For this we could just use collections.deque which is documented to be thread safe: https://docs.python.org/2/library/collections.html#collections.deque

Reply:
Done here an in other similar places in this file.